src/api/libcellml/issue.h

```diff
@@ -195,21 +195,27 @@ class LIBCELLML_EXPORT Issue
         MATH_CN_UNITS,
         MATH_CN_BASE10,
         MATH_CN_FORMAT,
+        ENCAPSULATION_ATTRIBUTE,
         ENCAPSULATION_CHILD,
         COMPONENT_REF_COMPONENT,
         COMPONENT_REF_CHILD,
+        CONNECTION_ATTRIBUTE,
         CONNECTION_COMPONENT1,
         CONNECTION_COMPONENT2,
         CONNECTION_EXCLUDE_SELF,
         CONNECTION_UNIQUE,
         CONNECTION_CHILD,
+        MAP_VARIABLES_ATTRIBUTE,
         MAP_VARIABLES_VARIABLE1,
         MAP_VARIABLES_VARIABLE2,
         MAP_VARIABLES_UNIQUE,
         MAP_VARIABLES_AVAILABLE_INTERFACE,
 
         // Issues not present in the normative specification:
         MAP_VARIABLES_IDENTICAL_UNIT_REDUCTION,
+
+        // Placeholder for further references:
+        NOT_SPECIFIED
```

Comment:
I would call it `UNSPECIFIED` rather than `NOT_SPECIFIED`, so that we are being consistent with `UNDEFINED` (rather than `NOT_DEFINED`).

If agreed, then I am happy to update it.

Reply:
Done in commit e5943638.